source/custom/nixnet_service.custom.cpp

```diff
@@ -1187,15 +1187,39 @@ void convert_to_grpc(std::vector<u8>& input, google::protobuf::RepeatedPtrField<
   }
 }
 
-void convert_to_grpc(const void* input, nixnet_grpc::FrameBuffer* output, u32 frame_type)
+void convert_to_grpc(const void* input, nixnet_grpc::FrameBufferResponse* output, u32 protocol, std::map<std::int32_t, std::int32_t> enetflags_output_map)
 {
-  if (frame_type == nixnet_grpc::FrameType::FRAME_TYPE_ENET) {
-    nixnet_grpc::EnetFrame* enet_frame = new nixnet_grpc::EnetFrame();
+  if (protocol == nixnet_grpc::Protocol::PROTOCOL_ENET) {
```

Comment:
Since these `if/else` blocks are much bigger now, it would be helpful to extract those into private helper methods `convert_frame_to_grpc` and `convert_enet_frame_to_grpc`

Reply:
Extracted into helpers